src/shared/lib/StoreUtils.ts

```diff
@@ -1251,3 +1252,25 @@ export async function getHierarchyData(
         sampleListId,
     });
 }
+
+export function currentReferenceGenomeBuildUrl(studies: CancerStudy[]) {
```

Comment:
this is specific to genome nexus, so since this util function lives in a general namespace (not genome nexus specific), maybe good to use something like `currentGenomeNexusReferenceGenomeBuildUrl`